packages/apps/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -9,14 +9,15 @@ import SpacingComponent from '@weco/common/views/components/SpacingComponent/Spa
 import SectionHeader from '@weco/common/views/components/SectionHeader/SectionHeader';
 
 import { AccountCreated } from './AccountCreated';
-import { AccountValidated } from '../AccountManagement/AccountValidated';
+import { AccountValidated } from './AccountValidated';
 import { ErrorMessage } from '../Shared/ErrorMessage';
 
 const logo = 'https://identity-public-assets-stage.s3.eu-west-1.amazonaws.com/images/wellcomecollections-150x50.png';
 import styled from 'styled-components';
 
 // At least 8 characters, one uppercase, one lowercase and number
 const passwordPolicy = /(?=.{8,})(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9]).*/;
+const emailTest = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Could we use an `<input type="email" />` and let the browser validate the email? I'm not sure how we can get the validity of the input to show a message, so perhaps it's not possible but there's a small risk that the regex may not work on all valid emails.

Reply:
We're also using the browser input type here, just using the regex as a double check to display message to user if we flag it!